logs/lib/getLogs.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+module.exports = {
+
+  getLogs() {
+    return BbPromise.bind(this)
+      .then(() => this.getNamespaceFromList(this.namespaceName))
+      .then((namespace) => this.listFunctions(namespace.id))
+      .then(this.getApplicationId)
+      .then(this.getLines)
+      .then(this.printLines)
+  },
+
+  getApplicationId(functions){
+    for (const fn of functions) {
+        if (fn.name == this.options.f) {
```

Comment:
this.options.f => What is it ? Use a more complete variable name please

Reply:
this is the function name passed with logs command:
`logs -f {function_name}`

Reply:
Yes I meant to rename this parameters to `functionName` for example (f is a bit too short, not easy to understand what it is)

Reply:
`-f` is required by serverless framework